src/game/Game.cpp

```diff
@@ -43,13 +43,13 @@ Game::Game()
   /* initialize Game class variables */
   isComplete = false;
   player.setCurrentRoom(0);
-  player.setXPos(2);
-  player.setYPos(2);
+  player.setXPos(12);
+  player.setYPos(12);
 
   /* initialize Rooms */
 
-  // room 0
-  rooms[0] = initStartRoom();
+  rooms[0] = initPuzzleRoom9();
+  rooms[1] = initStartRoom();
   rooms[2] = initPuzzleRoom2();
```

Comment:
put rooms[0] back to initStartRoom() and add a new line that reads:

rooms[9] = initPuzzleRoom9();